### ISSUE TITLE
Table Captions Not Being Tagged for Screen Readers

### ISSUE THREAD

**Issue by andreas-stefik** [borisveytsman/acmart#595]
I'm using the latest version of the template, which to my knowledge is:

[2026/06/27 v2.19 Typesetting articles for the Association for Computing Machinery]

To compile, I have been using overleaf and have tried two versions of the compiler:
2025
Rolling Texlive Labs

Problem:
If I create a table and give it a caption, screen readers are unable to access the value. This is testable with Voice Over by pressing the Voice Over Key (VO) + U. On my machine, that's CONTROL + OPTION + U after turning it on on Mac with COMMAND + F5. When you do so, you can traverse that rotor and go to the Table Menu with the right/left arrows. Seemingly no matter how I try to tag the caption, it never seems to work.

I've tried a variety of things. This includes different table structures (e.g., table*, removing placement like [H]), manually trying to inject my own tags, \Description, but to no avail. It's possible there's a way to do it that I'm missing.

In the rendered document for the reproducible example below, for example, the rotor reports from this template as:

<table> on Page 1. 

The two issues are:

- <table> should be something else, like a name or label. For the example below, my expectation would be for it to be "This Caption should be accessible."
- It says the table is on page 1, but it is on page 2. This is a minor issue, but gets pretty confusing for large documents.

Here's a small example generated from the common template that reproduces the issue:

\DocumentMetadata{tagging=on,
    tagging-setup={math/setup=mathml-SE},
    pdfstandard=ua-2,
    lang=en-US
}
\documentclass[acmsmall]{acmart}
\AtBeginDocument{%
  \providecommand\BibTeX{{%
    Bib\TeX}}}
\setcopyright{acmlicensed}
\copyrightyear{2018}
\acmYear{2018}
\acmDOI{XXXXXXX.XXXXXXX}
\acmConference[Conference acronym 'XX]{Make sure to enter the correct
  conference title from your rights confirmation email}{June 03--05,
  2018}{Woodstock, NY}
\acmISBN{978-1-4503-XXXX-X/2018/06}
\begin{document}
\title{The Name of the Title Is Hope}

\author{Ben Trovato}
\authornote{Both authors contributed equally to this research.}
\email{trovato@corporation.com}
\orcid{1234-5678-9012}
\author{G.K.M. Tobin}
\correspondingauthor
\authornotemark[1]
\email{webmaster@marysville-ohio.com}
\affiliation{%
  \institution{Institute for Clarity in Documentation}
  \city{Dublin}
  \state{Ohio}
  \country{USA}
}

\begin{abstract}
 Stuff
\end{abstract}

\maketitle

\section{Introduction}
Table is next.

\begin{table}
\centering
\caption{This Caption should be accessible.}
\tagpdfsetup{table/header-rows={1}}
\begin{tabular}{l l c c c c}
\toprule
\textbf{Software Design} & \textbf{Gender} & \textbf{Demographics} & \textbf{Completed First Task} & \textbf{Completed Last Task}\\
 \midrule
Experimental & Female             & 1  & 2  & 3\\  
             & Male               & 4 & 5  & 6\\  
             & Other              & 7   & 8   & 9\\  
             & Prefer not to say  & 10   & 11   & 12\\ \hline
Traditional  & Female             & 13  & 14  & 15\\  
             & Male               & 16 & 17 & 18\\  
             & Other              & 19   & 20   & 21\\  
             & Prefer not to say  & 22   & 23   & 24\\ \hline
\textbf{Total} &                  & 25 & 26 & 27\\ 
\bottomrule
\end{tabular}
\label{tab:participants}
\end{table}

\end{document}

**Comment by Carminiumn** [comment]
If possible, could you share a screenshot of the tag tree of the table and elements around it? I suspect it's due to caption being enclosed by the table tag (i.e. tagged as a child of the table).
